wpilibcExamples/src/main/cpp/templates/templates.json

```diff
@@ -43,7 +43,7 @@
   },
   {
     "name": "RobotBase Skeleton (Advanced)",
-    "description": "Skeleton (stub) code for RobotBase",
+    "description": "Skeleton (stub) code for RobotBase - Not for competition use",
```

Comment:
It can be used for competition, we just don't recommend it.

Reply:
How is that different from the other templates marked not for competition?